glusterd2/xlator/actors.go

```diff
@@ -6,19 +6,36 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+//optionActors in-memory to store registered xlator type and OptionActor
```

Comment:
It is implicit that this is in-memory. A comment on a map should describe what the key and values are.